examples/simple-filter-xml-mqtt/main.go

```diff
@@ -55,8 +55,12 @@ func main() {
 		Topic:     "sampleTopic",
 	}
 
+	pair := transforms.KeyCertPair{
+		KeyFile:  "",
+		CertFile: "",
+	}
 	mqttConfig := transforms.NewMqttConfig()
-	mqttSender := transforms.NewMQTTSender(edgexSdk.LoggingClient, addressable, "", "", mqttConfig)
+	mqttSender := transforms.NewMQTTSender(edgexSdk.LoggingClient, addressable, &pair, mqttConfig)
```

Comment:
Remove 58-61 and pass nil. This example will never have these values set and as it is you are passing empty values.

Reply:
@lenny-intel I just wanted to leave an example here to show the user how to pass in cert and key even though it won't work as is. Or I just pass in nil and leave the code creating KeyCertPair in comment like:
/*
        // use KeyCertPair to pass in key/cert pair if necessary
        pair := transforms.KeyCertPair{
        KeyFile:  "PATH_TO_YOUR_KEY_FILE",
        CertFile: "PATH_TO_YOUR_CERT_FIEL",
    }

    mqttSender := transforms.NewMQTTSender(edgexSdk.LoggingClient, addressable, &pair, mqttConfig)
*/

Reply:
Ok, I see where you are going. @rsdmike and I agree to leave the code in with the values set to "PATH_TO.." as you have above (fix typo _FIEL). Then add a comment that this will cause an error to be logged when attempting to load the files and that passing nil is an alternative when never using auth.

Reply:
@lenny-intel Update the code just doing so. Let me know if you further comments.

Reply:
@davidmsh , Perfect! Thanks!